Python Examples/TestMonitor/ReadMe.md

```diff
@@ -0,0 +1,38 @@
+Test Monitor Results Example
+=================
+
+This is an python example demonstrating how to use the
+SystemLink Test Monitor API to create test results and steps.
+
+Running the Example
+-------------------
+
+1. Clone _or_ download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
+2. Install the [Python SDK](https://www.python.org/downloads/).
+
+To run the example, use the following command:
+
+```
+python <filename.py> <url> <api_key>
```

Comment:
```suggestion
python results.py <url> <api_key>
```
Can we explicitly mention the file name for create results example?

Reply:
In the next line, this was given